### DISCUSSION TITLE
Protection DRM sur OQEE

### DISCUSSION THREAD

**Discussion by MortuusFR** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1266] [General]
Bonjour,

Quelqu'un à une solution gratuite pour faire des captures vidéos sur PC et qui outrepasse les protections DRM sur le site FREE OQEE ?

Je partage quelques fois des vidéos sur une plateforme afin de réagir dessus, je ne peux pas enregistrer sur OQEE et ça devient pénible..

Merci infiniment

**Comment by joaopa00** [comment]
Ce genre de discussion est illégale.
